### DISCUSSION TITLE
Can this library support ios9+

### DISCUSSION THREAD

**Discussion by duxiaoyan** [rabbitmq/rabbitmq-objc-client#184] [General]
Hello，you descrided this library currently targets iOS 11+ and macOS 10.12+， but  my app need support ios9+，will there be any problems if I continue to use it？
Thanks！

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
It's not tested on iOS 9 and releases distributed via e.g. CocoaPods won't be built for iOS 9 but you can try (the library hasn't changed meaningfully in several years) and use a fork/Git clone dependency.
